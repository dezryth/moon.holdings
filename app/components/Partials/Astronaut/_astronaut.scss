#astronaut {
  position: fixed;
  bottom: -5px;
  right: 40px;

  h1 {
    font-size: em(32);
    position: absolute;
    bottom: 20px;
    right: 10px;
  }
}
