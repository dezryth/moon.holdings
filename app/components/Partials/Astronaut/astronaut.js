import React from 'react';

export default ({ logo }) => (
  <section id="astronaut">
    { logo === true ? <h1>MOON.HOLDINGS</h1> : null }
    <img src="static/astronaut.png" alt="astronaut" />
  </section>
);
