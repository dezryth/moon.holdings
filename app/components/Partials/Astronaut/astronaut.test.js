import {
  getComponentWrapper,
  testCommonComponentAttrs
} from 'utils/tests';

import Astronaut from './astronaut';

describe('<Astronaut /> component', () => {
  testCommonComponentAttrs(Astronaut);

  it('should contain a section with id astronaut', () => {
    const wrapper = getComponentWrapper(Astronaut);
    expect(wrapper.find('section#astronaut')).toBeDefined();
  });
});
