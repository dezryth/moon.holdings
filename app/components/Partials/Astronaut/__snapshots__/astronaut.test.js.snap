// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Astronaut /> component when rendering should render a component matching the snapshot 1`] = `
<section
  id="astronaut"
>
  <h1>
    MOON.HOLDINGS
  </h1>
  <img
    alt="astronaut"
    src="static/astronaut.png"
  />
</section>
`;
